src/bindings/python/src/pyopenvino/graph/node_output.hpp

```diff
@@ -137,8 +139,8 @@ void regclass_graph_Output(py::module m, std::string typestring)
     output.def_property_readonly("any_name", &ov::Output<VT>::get_any_name);
     output.def_property_readonly("names", &ov::Output<VT>::get_names);
     output.def_property_readonly("element_type", &ov::Output<VT>::get_element_type);
-    output.def_property_readonly("shape", &ov::Output<VT>::get_shape);
-    output.def_property_readonly("partial_shape", &ov::Output<VT>::get_partial_shape);
+    output.def_property_readonly("shape", &ov::Output<VT>::get_shape, py::return_value_policy::copy);
+    output.def_property_readonly("partial_shape", &ov::Output<VT>::get_partial_shape, py::return_value_policy::copy);
```

Comment:
Should there be a note about copy?

Reply:
We can add docstring for all properties in a separate pr.